tests/acceptance/stepDefinitions/sharingContext.js

```diff
@@ -31,12 +32,29 @@ When('the user types {string} in the share-with-field', function (input) {
 })
 
 Then('all users and groups that contain the string {string} in their name should be listed in the autocomplete list on the webUI', function (pattern) {
-  return client.page.filesPage().sharingAutoCompleteShallContainElementsWith(pattern)
+  return client.page.filesPage().getShareAutocompleteItemsList()
+    .then(itemsList => {
+      itemsList.forEach(item => {
+        if (!item.toLocaleLowerCase().includes(pattern.toLocaleLowerCase())) {
```

Comment:
lets not convert this to lowercase, we can test how usernames that were created with upper-cases do act